lib/call-session.js

```diff
@@ -864,6 +864,12 @@ Duration=${payload.duration} `
         if (!offerMedia.flags.includes('asymmetric')) offerMedia.flags.push('asymmetric');
         offerMedia.flags = offerMedia.flags.filter((f) => f !== 'media handover');
       }
+      const acceptCodecs = process.env.JAMBONES_ACCEPT_AND_TRANSCODE ?
+        process.env.JAMBONES_ACCEPT_AND_TRANSCODE :
+        process.env.JAMBONES_ACCEPT_G729 ? 'g729' : '';
+      offerMedia = { ...offerMedia, ...(acceptCodecs && this.req.locals.cdr.direction ==="outbound" && { codec: { mask: acceptCodecs, transcode: 'pcmu,pcma' } })};
```

Comment:
since this is the outbound app, wont `this.req.locals.cdr.direction ==="outbound"` always be true?

(Also, we prefer to use singlequotes in js code)

Reply:
also, its not clear to me exactly what this has to do with hold/resume not working.  Would a better description be that hold and resume are not working when the initial codec was negotiated as something other than g711?

Reply:
@davehorton on re-invite if webrtc/thirdparty carrier is sending JAMBONES_ACCEPT_AND_TRANSCODE that need to be transcribed to pcmu which is done during answer of the call , this was breaking on re-invite , hence it needs to be handled